frontend/e2e/database/resetDatabase.ts

```diff
@@ -0,0 +1,23 @@
+import DB from "./db"
+import * as fs from "node:fs"
+
+const connectToDB = new DB()
+
+async function establishDBConnection() {
+  try {
+    await connectToDB.getDBConnection()
+  } catch (error) {
+    console.log(
+      `---------> X Failed to connect to dataBase <--------- \n\n ${error}`,
+    )
+    process.exit(1)
+  }
+}
+
+async function resetDatabase() {
+  await establishDBConnection()
+  const sql = fs.readFileSync("e2e/database/initial_data.sql").toString()
```

Comment:
Todo: Could we implement this instead by reading the XML files? Not a fan of these huge SQL files, they're impossible to maintain. Also, do you think it would make sense to share the data between backend and E2E tests? Or do we think those will be different?